bootstrap.sh

```diff
@@ -3,6 +3,9 @@
 # replace .bashrc file from vagrant directory to ~ directory
 cp /vagrant/.bashrc .bashrc
 
+# preventing error "$'\r': command not found"
+sed -i 's/\r$//' .bashrc
```

Comment:
some times there is an issue after the .bash has been added to the git 
and he gives an error that prevent the vagrant from including this file,
this is the fix for that problem

Reply:
AFAICS (for now) this file does not contain any '\r'
In [1]: f = open("/home/khakimi/git/beyond/lets-meet/.bashrc", 'r+')
In [2]: data = f.read()
In [3]: data
Out[3]: '# .bashrc\n\n# Source global definitions\nif [ -f /etc/bashrc ]; then\n        . /etc/bashrc\nfi\n\n# User specific environment\nif ! [[ "$PATH" =~ "$HOME/.local/bin:$HOME/bin:" ]]\nthen\n    PATH="$HOME/.local/bin:$HOME/bin:$PATH"\nfi\nexport PATH\n\n# Uncomment the following line if you don\'t like systemctl\'s auto-paging feature:\n# export SYSTEMD_PAGER=\n\n# User specific aliases and functions\nif [ -d ~/.bashrc.d ]; then\n        for rc in ~/.bashrc.d/*; do\n                if [ -f "$rc" ]; then\n                        . "$rc"\n                fi\n        done\nfi\n\n# addeing aliases\nalias manage="pipenv run python manage.py"\nalias shell="manage shell"\nalias run="manage runserver 0.0.0.0:8000"\nalias makemigrations="manage makemigrations"\nalias migrate="manage migrate"\nalias migration="makemigrations | migrate"\nalias run_tests="pipenv run pytest -v"\nalias run_flake8="pipenv run flake8 --max-line-length 120"\n\nalias find_program="netstat -plten | grep python"\nalias stop_program="sudo kill -9 "\n\nunset rc\n'

So I think this workaround is not needed 
in anyway I would suggest to fix the file once instead of run this command every time we run the bootstap

Reply:
I have already tries and filed to replace all \r before hand,
after git adds the file its been changed, 
this is not something I can test and not something I can try to change, this is how git add the file.

This is my solution to the problem I have encountered, and all of my teammates and this solution has helped all of them.

Reply:
As I have said, this problem can accrue when I uploaded the file,
this problem doesn't always show up but the line I have added fixed it.

I request that this line will stay any way because the problem can show up in any time and will cause problems.